src/feature/control/control_getinfo.h

```diff
@@ -48,6 +48,10 @@ STATIC int getinfo_helper_downloads(
     control_connection_t *control_conn,
     const char *question, char **answer,
     const char **errmsg);
+STATIC int getinfo_helper_current_consensus(
+    const char* consensus_type,
```

Comment:
Other code uses flavor_name
```suggestion
    const char* flavor_name,
```